app/src/main/report/components/Reports/ReportDetails.tsx

```diff
@@ -44,13 +44,15 @@ export class ReportDetailsComponent extends RemoteContentComponent<ReportDetails
     renderContent(props: ReportDetailsProps) {
         const version = props.versionDetails.id;
         return <div>
-           <DraftStamp published={props.versionDetails.published} />
+            <h1 className={"h2"}>{props.versionDetails.displayname || props.versionDetails.name}</h1>
+            <p className={"small text-muted"}>{props.versionDetails.id}</p>
```

Comment:
The report title has moved down as we're no longer using the `Page` component with integrated title